src/modules/claims/claims.service.ts

```diff
@@ -1494,12 +1505,32 @@ export class ClaimsService {
     if (!proofVerified) {
       errors.push(ERROR_MESSAGES.PROOF_NOT_VERIFIED);
     }
+    let isExpired = false;
+    if (payload?.exp) {
+      if (this.claimIsExpired(payload.exp)) {
+        isExpired = true;
+        errors.push(ERROR_MESSAGES.CLAIM_EXPIRED);
+      }
+    }
     return {
       errors: errors,
-      isVerified: !!proofVerified && !!issuerVerified,
+      isVerified: !!proofVerified && !!issuerVerified && !isExpired,
     };
   }
 
+  async fetchCredential(
```

Comment:
I created this method just so that I could grab the credential after I do 'requestAndEnrol', so that I have the cred object to pass to the verifyRoleEIP191JWT. I'm not sure how else I can get the object after its issued otherwise?